Wikipedia/Code/WMFTableViewCellWithCustomMinusButton.h

```diff
@@ -0,0 +1,11 @@
+#import <UIKit/UIKit.h>
+
+/**
+ * Table cell with a custom minus button achieved by overlaying the default minus button with a custom minus button.
+ */
+
+@interface WMFTableViewCellWithCustomMinusButton : UITableViewCell
+
+@property (strong, nonatomic, readonly) UIButton* minusButton;
```

Comment:
`deleteButton`

In general we should name things base don functionality rather than appearance as the designs can change but the purpose of deleting will stay the same.